src/Renraku/ReAbstractRule.class.st

```diff
@@ -74,7 +74,7 @@ ReAbstractRule class >> identifierMinorVersionNumber [
 { #category : #testing }
 ReAbstractRule class >> isUsed [
 	"all my sublasses are used"
-	^self name = 'RBLintRule'
+	^self name = 'ReAbstractRule'
 		ifTrue: [ super isUsed ]
 		ifFalse: [ true ]
 ]
```

Comment:
Why not

```Smalltalk
^self = ReAbstractRule
 		ifTrue: [ super isUsed ]
 		ifFalse: [ true ]
```

?

Reply:
because then the QA rule tells us that we should use "self", but *that* is wrong as it would not do the correct thing when called for a subclasses.

Reply:
I do not like it too much.. better ideas are welcome (the pattern is used everywhere where we have subclasses that are not referenced directly but used reflectively)

Reply:
maybe we can fix the lint rule to ignore all these methods?